azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/b2c/AADB2CAuthenticationProperties.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+package com.microsoft.azure.spring.autoconfigure.b2c;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.validator.constraints.URL;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+import javax.validation.constraints.NotBlank;
+
+@Getter
+@Setter
+@Validated
+@ConfigurationProperties(prefix = AADB2CAuthenticationProperties.PREFIX)
+public class AADB2CAuthenticationProperties {
```

Comment:
Do we really need `Authentication` in the class name?

Reply:
In previous, I prefer to remove this word from classname. But security contains both authentication and authorize, so I keep this word in case we may need authorize properties in future. If we can merge these 2 parts, then we can remove this words safely.